tests/test_nlu.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import nlu
+
+@pytest.fixture()
+def text():
+    return '我想查曠課'
+
+def test_nlu_execute(text):
+    assert nlu.execute(text) == '我想查, 曠課'
```

Comment:
Do not test external lib, at here what you should test should be can the NLU returns correct command you want to be run or what, if we can't do that right now then please remove the test, its annoying and useless at here

Reply:
agree